fifteen_min/src/find_home.rs

```diff
@@ -35,7 +35,7 @@ impl FindHome {
             )
             .flex_wrap(ctx, Percent::int(50)),
             ctx.style()
-                .btn_solid_dark_text("Search")
+                .btn_solid_text("Search")
```

Comment:
The thrust of the PR is changes like this, where we're no longer hardcoding dark/light into the UI components. 

This was made possible by discussing with Yuwen how we could simplify the design somewhat to use consistent components in the panels vs the top level map controls.